### PR TITLE
Add an email text option to the contact snippet

### PR DESCRIPTION
**Pull request by helenb**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1565899542)

### Description of Changes Made

- Add a new option to the contact CTA snippet to display an email address as a mailto link, separate to the cta button. We are leaving the option to add an email on the button itself until Lily has had a chance to update existing content.

### How to Test

- Add a contact snippet in your local build, then edit any page. In the promote panel, select your contact snippet.

### Screenshots

<details>
  <summary>Expand to see more</summary>

Screenshot 2024-07-19 at 12 34 16

Screenshot 2024-07-19 at 12 34 37

</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Tests and linting passes.

#### Unit tests

- [ ] Added
- [x] Not required

#### Documentation

- [x] Updated build docs
- [ ] Updated editor guidelines (See https://intranet.torchbox.com/torchbox-com-project-docs for the private link, for Torchbox employees only)
- [ ] Updated field spec (See https://intranet.torchbox.com/torchbox-com-project-docs for the private link, for Torchbox employees only)
- [ ] Not required

#### Browser testing

- [x] I have tested in the following browsers and environments (edit the list as required)
  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android
- [ ] Not required

#### Data protection

- [x] Not relevant
- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly

#### Accessibility

- [x] Automated WCAG 2.1 tests pass
- [x] HTML validation passes
- [x] Manual WCAG 2.1 tests completed
- [x] I have tested in a screen reader
- [x] I have tested in high-contrast mode
- [ ] Any animations removed for prefers-reduced-motion
- [ ] Not required

#### Sustainability

- [ ] Images are optimised and lazy-loading used where appropriate
- [ ] SVGs have been optimised
- [ ] Perfomance and transfer of data considered
- [ ] If JavaScript is needed alternatives have been considered
- [x] Not required

#### Pattern library

- [x] The pattern library component for this template displays correctly, and does not break parent templates
- [ ] The styleguide is updated if relevant
- [ ] Changes are not relevant the pattern library

### CODE REVIEW

File: tbx/static_src/sass/components/_footer-cta.scss [hunk 1]
```diff
@@ -67,4 +67,18 @@
...
+        margin-bottom: $spacer-small;
+
+        @include media-query(large) {
+            margin-bottom: 0;
+        }
+    }
+
+    &__email {
+        @include link-styles();
+        //@include focus-styles();
```
Comment (chris-lawton) [inline tbx/static_src/sass/components/_footer-cta.scss:82]: Looks like the focus styles are working without this so can probably be deleted : )

File: tbx/project_styleguide/templates/patterns/molecules/footer-cta/footer-cta.html [hunk 2]
```diff
@@ -13,6 +13,9 @@
 
         <div class="footer-cta__button-wrapper">
             <a href="{{ contact_link }}" class="button footer-cta__button">{{ contact_action }}</a>
+
+            <p class="footer-cta__email-wrapper"><a class="footer-cta__email" href="mailto:{{ contact_email }}">{{ contact_email }}</a></p>
```
Comment (chris-lawton) [inline tbx/project_styleguide/templates/patterns/molecules/footer-cta/footer-cta.html:17]: Question: Do we need to wrap this in a conditional as the field in Wagtail that populates it isn't required?

**Review by chris-lawton — APPROVED** [review]
Working as expected @helenb - thanks!

File: tbx/project_styleguide/templates/patterns/molecules/footer-cta/footer-cta.html (hunk 2 above)
Comment (helenb) [reply tbx/project_styleguide/templates/patterns/molecules/footer-cta/footer-cta.html:17]: good point will fix that.

File: tbx/static_src/sass/components/_footer-cta.scss (hunk 1 above)
Comment (helenb) [reply tbx/static_src/sass/components/_footer-cta.scss:82]: whoopsy...

**Review by nickmoreton — APPROVED** [review]
Hi @helenb Thanks looks good and works as described.

**Pull request merged**